src/Mod/Start/App/CustomFolderModel.cpp

```diff
@@ -39,30 +40,46 @@ CustomFolderModel::CustomFolderModel(QObject* parent)
     Base::Reference<ParameterGrp> parameterGroup = App::GetApplication().GetParameterGroupByPath(
         "User parameter:BaseApp/Preferences/Mod/Start");
 
-    _customFolderDirectory =
-        QDir(QString::fromStdString(parameterGroup->GetASCII("CustomFolder", "")));
+    _customFolderPathSpec = QString::fromStdString(parameterGroup->GetASCII("CustomFolder", ""));
 
     _showOnlyFCStd = parameterGroup->GetBool("ShowOnlyFCStd", false);
 }
 
+/// If the custom folder path contains multiple paths separated by ';;', split them into individual
+/// paths. This is used to allow the user to specify multiple paths in the preferences dialog.
+/// We use ';;' as a separator because ';' is a valid character in a file path (e.g. NTFS on
+/// Windows).
 void CustomFolderModel::loadCustomFolder()
 {
     beginResetModel();
     clear();
-    if (!_customFolderDirectory.isReadable()) {
-        Base::Console().Warning(
-            "BaseApp/Preferences/Mod/Start/CustomFolder: cannot read custom folder %s\n",
-            _customFolderDirectory.absolutePath().toStdString().c_str());
-    }
+    auto pathDelimiter = QStringLiteral(";;");
+    auto paths = _customFolderPathSpec.split(pathDelimiter, Qt::SkipEmptyParts);
 
-    if (_showOnlyFCStd) {
-        _customFolderDirectory.setNameFilters(QStringList() << QStringLiteral("*.FCStd"));
-    }
+    for (const auto& path : paths) {
+        QDir customFolderDirectory(path);
+        if (!customFolderDirectory.exists()) {
+            Base::Console().Warning(
+                "BaseApp/Preferences/Mod/Start/CustomFolder: custom folder %s does not exist\n",
+                customFolderDirectory.absolutePath().toStdString().c_str());
+            continue;
+        }
+        if (!customFolderDirectory.isReadable()) {
+            Base::Console().Warning(
+                "BaseApp/Preferences/Mod/Start/CustomFolder: cannot read custom folder %s\n",
+                customFolderDirectory.absolutePath().toStdString().c_str());
+            continue;
+        }
+        if (_showOnlyFCStd) {
+            _customFolderDirectory.setNameFilters(QStringList() << QStringLiteral("*.FCStd"));
```

Comment:
```suggestion
            customFolderDirectory.setNameFilters(QStringList() << QStringLiteral("*.FCStd"));
```

The merge went wrong here.